.github/workflows/pr.yml

```diff
@@ -84,6 +84,12 @@ jobs:
             git --no-pager diff ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen'
             exit 1
           fi
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
+        with:
+          args: ./...
+        env:
+          GOROOT: ""
```

Comment:
For my understanding. Why you need to change the GOROOT?

Reply:
Using gosec in GitHub action seems to need GOROOT unset.
See here for specific details:
https://github.com/securego/gosec/issues/469